fatiando/mesher/geometry.py

```diff
@@ -92,6 +94,57 @@ def x(self):
     def y(self):
         return self.vertices[:, 1]
 
+    @property
+    def orientation(self):
+        area = self.get_area(absolute=False)
+        if area < 0:
+            return "CW"
+        else:
+            return "CCW"
```

Comment:
Please change `CW` and `CCW` to `clockwise` and `counterclockwise` for readability.